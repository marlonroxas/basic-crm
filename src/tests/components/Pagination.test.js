import React from 'react'
import { shallow } from 'enzyme'

import Pagination from '../../components/Pagination/index'

test('Should render Pagination correctly', () => {
  const wrapper = shallow(<Pagination />)
  expect(wrapper).toMatchSnapshot()
})