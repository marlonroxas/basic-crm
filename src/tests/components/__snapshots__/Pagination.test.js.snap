// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Pagination correctly 1`] = `
<ul
  className="pagination"
/>
`;
